processors/user-group.ftd

```diff
@@ -0,0 +1,31 @@
+-- ft.page: `user-groups` processor
+
+`user-groups` processor can be used to find all the [groups](/dev/user-group/)
+the current user belongs to.
```

Comment:
This is incorrect. This processor returns all the groups defined in FPM.ftd, not the groups current logged in user belongs to.